### ISSUE TITLE
Checking Stack update progress... never completes

### ISSUE THREAD

**Issue by nwhite89** [serverless/serverless#2691]
# Bug Report

## Description

For bug reports:
* What went wrong?

When running deploy I can't get past `Serverless: Checking Stack update progress…`. Despite the CloudFormation in AWS stating that it is `UPDATE_COMPLETE`

* What did you expect should have happened?

Update with completion from `CloudFormation` and Service Information

* What was the config you used?

```
provider:
  name: aws
  runtime: nodejs4.3
  region: eu-west-1
  deploymentBucket: lambda-test-formats
  cfLogs: true
package:
  exclude:
    - .reports/**
    - .git/**
    - api/**
    - dist/**
    - grunt/**
    - test/**
functions:
  demos:
    handler: handler.demos
    events:
      - http:
          path: demos
          method: get
  publish:
    handler: handler2.publish
    events:
      - http:
          path: publish
          method: get
```

* What stacktrace or error message from your provider did you see?
N/A

Similar or dependent issues:

## Additional Data

* ***Serverless Framework Version you're using***: 1.10
* ***Operating System***: OSX 10.11.4
* ***Stack Trace***: N/A
* ***Provider Error messages***: N/A

![screen shot 2016-11-11 at 10 20 30 1](https://cloud.githubusercontent.com/assets/1407863/20212220/df2916c0-a7f9-11e6-9305-24f9b9d9c82f.png)

**Comment by marjanSterjev** [comment]
Check your system clock. It is probably behind. Setup the clock ( manually or ntp) and try again. This was reported previously and postponed as non critical.

**Comment by nwhite89** [comment]
I got a bit fed up with it as it was working with a colleagues machine on Macon Sierra aftrr upgrading it worked I can't imagine it would be the OSX issue so more than likely simply restarting would have fixed it. @marjanSterjev thanks will close this issue

**Comment by bobby-brennan** [comment]
FYI - I just hit this bug as well. A reboot fixed the issue.

I would suggest reopening this issue as non-critical. That will make the workaround instructions more visible (I spend a few hours tearing my hair out before thinking to check closed issues).  As multiple people seem to be hitting this it's probably worth investigating further.

**Comment by nicolasblanco** [comment]
I'm having this problem. I tried to reboot, checked the clock...
It's stuck at `Serverless: Checking Stack update progress...`.

No other information even when using verbose mode. How can I debug this?

**Comment by pmuens** [comment]
Hey @nicolasblanco thanks for reporting that it's unfortunately still an issue for you.

What Node version are you using?

Maybe you can also take a quick look [here](https://github.com/serverless/serverless/issues/2370) where the same problem is discussed.

**Comment by nicolasblanco** [comment]
I tried with both LTS (6.9.5) and latest (7.5.0).
It's not a Node problem, I've tried to debug and found out that `eventInRange` in the `monitorStack` loop was always `false` : https://github.com/serverless/serverless/blob/master/lib/plugins/aws/lib/monitorStack.js#L46

My OS X was set with automatic clock, I set that to off then manually set my clock 5 minutes backward, then it worked. Does it mean that for some reason my clock was too ahead of the Amazon servers?
